ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml

```diff
@@ -122,17 +124,17 @@ objects:
         # TODO: bump nodes up to 40 again
         source /tmp/shared/overrides
         set -x
-        if [[ -n "${TEST_FOCUS}" ]]; then
+       if [[ -n "${TEST_SUITE}" || -n "${TEST_FOCUS}" ]]; then
           ginkgo -v -noColor -nodes=30 $( which extended.test ) -- \
-            -ginkgo.focus="${TEST_FOCUS}" -ginkgo.skip="${TEST_SKIP}" \
+            -suite "${TEST_SUITE}" -ginkgo.focus="${TEST_FOCUS}" -ginkgo.skip="${TEST_SKIP}" \
             -e2e-output-dir /tmp/artifacts -report-dir /tmp/artifacts/junit \
-            -test.timeout=2h || rc=$?
+            -test.timeout=2h ${PROVIDER_ARGS-} || rc=$?
```

Comment:
Why `PROVIDR_ARGS`?  If you want to consume it, you should probably set it.  For example:

```console
$ git grep PROVIDER_ARGS ci-operator/templates/cluster-launch-installer-e2e.yaml
ci-operator/templates/cluster-launch-installer-e2e.yaml:          export PROVIDER_ARGS='-provider
gce -gce-zone=us-east1-c -gce-project=openshift-gce-devel-ci'
ci-operator/templates/cluster-launch-installer-e2e.yaml:          export PROVIDER_ARGS="-provider
aws -gce-zone=${region}"
ci-operator/templates/cluster-launch-installer-e2e.yaml:            -test.timeout=10m ${PROVIDER_
RGS-} || rc=$?
ci-operator/templates/cluster-launch-installer-e2e.yaml:            -test.timeout=20m ${PROVIDER_
RGS-} || rc=$?
```

But if you don't have providers to distinguish between, it seems simpler to just remove `PROVIDER_ARGS` entirely.

Reply:
> Azure dont use it, so we remove it so not to confuse

So remove your addition here?

Reply:
done